ci/vm/entrypoint.sh

```diff
@@ -45,14 +49,19 @@ xpmem_build() {
 
 xpmem_load() {
   sudo insmod kernel/xpmem.ko
-  if ! lsmod | grep -q xpmem; then
+  if modinfo kernel/xpmem.ko | grep -qf /sys/module/xpmem/srcversion; then
```

Comment:
maybe run both commands (modinfo kernel/xpmem.ko and cat /sys/module/xpmem/srcversion) before this if/else so in case of error we could see the problem in the log

Reply:
> @yosefe
> Is there a point to still build on Docker, since we both build and test on VMs?

@Alexey-Rivkin I think better build on docker since we'll eventually have more OS versions to build on than to run tests on

Reply:
Or we can get output of both commands in case of error, WDYT?